Rakefile

```diff
@@ -292,16 +271,6 @@ namespace :install do
     end
   end
 
-  task :shared do
-    if local_environment?
-      Dir.chdir(shared_js_dir) do
-        shared_js_build = CDO.use_my_shared_js ? shared_js_dir('build/package') : 'shared-package'
-        RakeUtils.ln_s shared_js_build, dashboard_dir('public','shared')
```

Comment:
shared-package is part of the repo, so I think that will just go away. The symlink is a more interesting question. There shouldn't be any harm in having an extra symlink there. I could provide a script for devs to run, though all it would do is delete the symlink (so I could also just tell devs to do that)